src/node_crypto.cc

```diff
@@ -3462,6 +3462,10 @@ Local<String> KeyObject::GetAsymmetricKeyType() const {
     return env()->crypto_dsa_string();
   case EVP_PKEY_EC:
     return env()->crypto_ec_string();
+  case EVP_PKEY_ED25519:
+    return env()->crypto_ed25519_string();
+  case EVP_PKEY_ED448:
+    return env()->crypto_ed448_string();
```

Comment:
I believe I explicitely mentioned `'rsa'`, `'dsa'` and `'ec'` at some places in the documentation, these might need to be updated.

Reply:
Added.